pontojson.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+"""
+pontojson.py: Convert Horrible People Games' cards.pon files to .json format
+For use with the TSSSF Card Generator module
+"""
+__author__ = "Jouva Moufette"
+__copyright__ = "Copyright 2016, Children of Kefentse"
+__credits__ = ["", ""]
+__license__ = "CC-BY-SA"
+__version__ = "1.0.0"
+__maintainer__ = "Jouva Moufette"
+__email__ = "jouva@moufette.com"
+
+import sys
+import json
+import os
+import distutils.version
+from collections import OrderedDict
+
+TYPE, PICTURE, SYMBOLS, TITLE, KEYWORDS, BODY, FLAVOR, EXPANSION, CLIENT = range(9)
+DEFAULT_ARTIST = "Pixel Prism"
+
+Expansion_Icons = {
+    "Indiegogo": "symbol-Indiegogo.png",
+    "Summer": "symbol-summer-lovin.png",
+    "Xtra": "symbol-extracredit.png",
+    "Adventure": "symbol-adventure.png",
+    "Custom": "symbol-custom.png",
+    "Power": "symbol-power.png",
+    "Multiplicity": "symbol-multiplicity.png",
+    "Canon": "symbol-canon.png",
+    "Dungeon": "symbol-dungeon.png",
+    "2014": "symbol-2014.png",
+    "2015": "symbol-2015.png",
+    "Workshop": "symbol-workshop.png",
```

Comment:
Is this needed, as there hasn't been a Workshop symbol in any of the current sets, or is this for future stuff?